libats-auth/src/main/scala/com/advancedtelematic/libats/auth/NamespaceDirectives.scala

```diff
@@ -0,0 +1,46 @@
+package com.advancedtelematic.libats.auth
+
+import akka.http.scaladsl.model.HttpHeader
+import akka.http.scaladsl.model.headers.RawHeader
+import akka.http.scaladsl.server.Directive1
+import com.advancedtelematic.jwt.JsonWebToken
+import com.advancedtelematic.libats.data.Namespace
+import com.typesafe.config.{Config, ConfigFactory}
+import org.slf4j.LoggerFactory
+
+import scala.util.Try
+
+object NamespaceDirectives {
+  import akka.http.scaladsl.server.Directives._
+
+  lazy val logger = LoggerFactory.getLogger(this.getClass)
+
+  val NAMESPACE = "x-ats-namespace"
+
+  def nsHeader(ns: Namespace): HttpHeader = RawHeader(NAMESPACE, ns.get)
+
+  def configNamespace(config: Config): Namespace = {
+    Namespace( Try(config.getString("core.defaultNs")).getOrElse("default"))
```

Comment:
This config lookup should probably not be here, (or at least not lookup in `core.defaultNs`